scripts/dataset_yaml2js.js

```diff
@@ -1,5 +1,6 @@
 var fs = require('fs')
   , util = require('util')
+  , serialize = require('serialize-to-js')
```

Comment:
Use `serialize_to_js` or something to show it's not a general thing, but a library.